default/scripting/ship_hulls/ship_hulls.macros

```diff
@@ -202,13 +202,13 @@ HULL_FUEL_EFFICIENCY_EFFECTSGROUP
      EffectsGroup
             description = "MAX_FUEL_LESS_THAN_ONE_DESC"
             scope = And [
-                Ship
                 Source
-                MaxFuel high = 0.9999
+                Ship
             ]
             accountinglabel = "MAX_FUEL_LESS_THAN_ONE_LABEL"
             priority = [[METER_OVERRIDE_PRIORITY]]
-            effects = SetMaxFuel value = 0
+            // Have to use Value() in effects, as all scope and activation conditions are evaluated before applying any effect
```

Comment:
This comment doesn't make much sense outside the context of this pull request and related discussion...  Maybe write what you're doing first, then that that has to be done with a test / filter in the effect execution and not in the activation or scope because no effects have not yet been applied yet at the time the scope and activation are evaluated.